cura/Settings/ExtruderManager.py

```diff
@@ -78,6 +78,7 @@ def extruderCount(self):
     def extruderIds(self):
         map = {}
```

Comment:
Just noticed this; But map shadows a built in. Could this be re-factored to a better name?